### DISCUSSION TITLE
Cron isse: Want to run every 32 minutes, but runs at top of the hour + top of the hour and 31 minutes.

### DISCUSSION THREAD

**Discussion by fischerphilipp** [alexjustesen/speedtest-tracker#1359] [Q&A] [answered]
Hi Alex,

thanks a lot for creating and maintaining this tool, really helpful for me.

When configuring and adjusting the speedtests I discovered the following:
I initially had the speedtests run at the top of the hour, like 8:00, 9:00 etc. However apparently some speedtest servers are at maximum capacity at that time because I discovered that when there are breaches to the set thresholds, this almost always occurs at the top of the hour. If I move the tests to a little later, then the results are way more consistent and reaching my ISP top speed. 

My current Cron setting is */32 * * * * 
However I have two issues with the tests:
* Although I set it to run at /32 I always also get a test at the the top of the hour. No idea where this is coming from (see screenshot).
* The cron generator you linked says that /32 should let the test run every 32 minutes (which is what I want), instead I see the tests running always at 32 minutes past the hour. Because I want ~two tests within one hour I would prefer it to run as the generator says. Is that an issue with the linked cron generator or with Speedtest tracker?

Thanks in advance and best regards,
Philipp
![Screenshot 2024-04-08 at 10 03 06](https://github.com/alexjustesen/speedtest-tracker/assets/68103956/f992742a-8f0f-4d96-acad-97a34eb58975)

**Comment by alexjustesen** [comment, marked as answer]
https://github.com/alexjustesen/speedtest-tracker/issues/552#issuecomment-2028532010
